app/assets/stylesheets/consul.scss

```diff
@@ -35,14 +30,19 @@
       margin-top:4em;
     }
 
-    p{
-      margin: 0 auto;
+    p, ul{
```

Comment:
After this change the `ul` gets a top margin it didn't previously have. Due to that, the top of the list of members isn't aligned anymore with the top of the other column. Is that intentional?